systest/testcontext/context.go

```diff
@@ -239,6 +243,8 @@ func New(t *testing.T, opts ...Opt) *Context {
 		TestID:            *testid,
 		Keep:              *keep,
 		ClusterSize:       *clusterSize,
+		PoetSize:          *poetSize,
```

Comment:
needs to be saved in labels like keep and cluster size

Reply:
done